go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	github.com/kr/pretty v0.3.1
 	github.com/moby/moby v20.10.20+incompatible
 	github.com/stretchr/testify v1.8.0
+	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
```

Comment:
https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap seems to be the official package from the kernel. Is it possible to use it?

Reply:
This is also true that https://github.com/syndtr/gocapability is used by a lot of projects, so up to you which one to use.

Reply:
syndtr/gocapability  is also used in runc.

The other lib has a String() method but it seems only have to describe all the full capability sets (effective, inheritable, etc.). That could be useful to add in a future version of the gadget but for now I only give the effective set since it is the set used by the kernel to check the capability.

So I will keep syndtr/gocapability for now.